### PR TITLE
feat(config): add Aeotec Range Extender 7, add id for Fibaro Heat controller

### PR DESCRIPTION
**Pull request by varet80**
Added the following:

- New product ID for Fibaro Heat controller
- New product config for AEOTEC Range Extender 7

### CODE REVIEW

**Comment by robertsLando** [comment]
Ref #1218

File: packages/config/config/devices/0x0371/zwa189.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+			"productId": "0x00bd"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+	}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa189.json]: Please remove this

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
Tell me when you're finished, then I'll approve and merge.

**Comment by varet80** [comment]
I removed the paramInformation and fixed all minor typos!  @AlCalzone

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
This is going into 5.6.1

**Pull request merged**